src/main/java/cinemas/controllers/user/PaymentsController.java

```diff
@@ -0,0 +1,16 @@
+package cinemas.controllers.user;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/payments/{id}")
```

Comment:
```suggestion
@RequestMapping("/payments/{id}")
```
nếu chị ko lầm thì id nì là user id nhỉ?
nếu đúng thì nó nên nằm trước payments nhé, và nên ghi rõ là user_id chứ ko phải id
vì id ở đây là id của payments đó
```suggestion
@RequestMapping("{user_id}/payments")
```